docs/2026-06-25_aws_gateway_benchmark/remote/run-on-instance.sh

```
@@ -0,0 +1,379 @@
+#            under sustained load.
+#
+# Results are written as JSON to ./results/ for the orchestrator to collect.
+#
+# NOTE: deliberately NOT `set -e`. This is a resilient benchmark harness — a
+# single flaky docker/compose/curl on one variant must not abort the whole run;
+# it should skip to the next variant and still reach the final meta.json sentinel
+# the orchestrator polls for. Failures are visible in each variant's ok/failed.
+set -uo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+cd "$SCRIPT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)